commit 7ff9e82dac45facb9d807ccef0f506fba20f649a
Author: Valdis Vitolins <valdis.vitolins@odo.lv>
Date:   Sun Dec 6 21:04:26 2015 +0200

    https://sourceforge.net/p/espeak/bugs/126/
    the relocations produced by -fpic are not enough, so -fPIC should be used instead, in src/Makefile

diff --git a/src/Makefile b/src/Makefile
index 17224b8..fc70ebb 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -110,7 +110,7 @@ $(BIN2_NAME): $(OBJS3) $(LIB_NAME)
 
 
 x_%.o: %.cpp
-	$(CXX) $(CXXFLAGS) $(USE_AUDIO) -D PATH_ESPEAK_DATA=\"$(DATADIR)\" -Wall -fpic -fvisibility=hidden -pedantic \
+	$(CXX) $(CXXFLAGS) $(USE_AUDIO) -D PATH_ESPEAK_DATA=\"$(DATADIR)\" -Wall -fPIC -fvisibility=hidden -pedantic \
 	-I. -D LIBRARY -c -fno-exceptions  $<  -o x_$*.o
 
 $(LIB_NAME): $(OBJS2)
